content/en/docs/ops/diagnostic-tools/istioctl-analyze/test.sh

```diff
@@ -51,7 +51,7 @@ echo '*** istioctl-analyze step 7 ***'
 istioctl analyze --help
 
 echo '*** istioctl-analyze step 8 ***'
-snip_install_with_custom_config_analysis
+echo y | snip_install_with_custom_config_analysis
```

Comment:
Why are you not using the `yes` command for this?
```suggestion
yes | snip_install_with_custom_config_analysis
```

Reply:
Lack of knowing the command was there. Will pick up change(s) in cleanup commit now that tests are working.

Reply:
Tried the change to `yes |` and the tests fail:
```
2020-09-16T15:23:24.676474Z	info	tf	Running command script tasks_security_authorization_authz-td-migration_test
    script.go:111: script tasks_security_authorization_authz-td-migration_test returned an error: exit status 141. Output:
        This will install the Istio profile into the cluster. Proceed? (y/N) 
        - Processing resources for Istio core.
        ✔ Istio core installed
        - Processing resources for Istiod.
        - Processing resources for Istiod. Waiting for Deployment/istio-system/istiod
        ✔ Istiod installed
        - Processing resources for Egress gateways, Ingress gateways.
        - Processing resources for Egress gateways, Ingress gateways. Waiting for Deployment/istio-system...
        ✔ Egress gateways installed
        ✔ Ingress gateways installed
        - Pruning removed resources
        ✔ Installation complete
```
Trying the command locally yields a SIGPIPE(13) failure.

Reply:
Ah, so it's probably failing because we run the tests with `set -o pipefail`. I don't know of that's really needed.

Reply:
I'm not sure about changing the pipefail. Can we do it file wide? Or is this something we would change around each invocation? For now, `echo y` is being used and working.

Reply:
Not worth changing it around every invocation, better to use `echo y` in that case, but maybe we could change it file wide. It's just a convention we've been using in the test.sh files, but I'm not sure if it's needed or not. Anyway, another thing to investigate when we have some free time.